### ISSUE TITLE
Icon appearance in OSX is displayed wrong with Mozilla Mix from CTB or old OSX

### ISSUE THREAD

**Issue by TW462** [Aris-t2/CustomCSSforFx#369]
1. Describe the issue
With Firefox 89 on OSX, navigation buttons for "refresh", "home", "search", and "print" has a squeezed appearance

2. What should have happened instead?
The OSX navigation buttons should be the same size

3. Steps to reproduce the issue?
Enabling a different icon appearance changes accordingly

4. SCREENSHOTS with/without this projects files (drag & drop images into this post)

![image](https://user-images.githubusercontent.com/77515829/120899793-52f6fb80-c5e6-11eb-83fc-9ec7850eb894.png)

![image](https://user-images.githubusercontent.com/77515829/120899827-79b53200-c5e6-11eb-8d8d-34cf1e33656a.png)

![image](https://user-images.githubusercontent.com/77515829/120899883-b6812900-c5e6-11eb-8e05-102401261cfd.png)

![image](https://user-images.githubusercontent.com/77515829/120899871-a701e000-c5e6-11eb-8d88-ff9d32b927fc.png)

![image](https://user-images.githubusercontent.com/77515829/120899905-d9abd880-c5e6-11eb-960b-3adaf71d3495.png)

![image](https://user-images.githubusercontent.com/77515829/120899915-e4ff0400-c5e6-11eb-82ab-5c7c10e340e8.png)

5. System information
- latest CustomCSSforFx files (yes / no [no support, if 'no']): yes
- OS & OS version: OSX 11.4
- OS theme (if not default): 
- DPI / HiDPI resolution: 1680x1050
- Firefox version: 89
- Firefox theme / lw-theme: Default
- This projects 'settings' the issue occurs with: 3.2.1v2

**Comment by Aris-t2** [comment]
https://github.com/Aris-t2/CustomCSSforFx/releases/tag/3.2.2

**Comment by TW462** [comment]
Hi Aris-t2,

Will use the latest files you sent me a link to.

Thank you once again for your help.

Thomas

**Comment by Aris-t2** [comment]
By now the latest file is this one: https://github.com/Aris-t2/CustomCSSforFx/files/6604433/chrome.zip

**Comment by TW462** [comment]
Hi Aris-t2,

I am using your latest version 3.2.4. The only thing I have noticed is that the active tab shows a gap with the active tab color above the main body with the background color being displayed. Please see attached screen shot. I will look through the files and see what I did not enable or not disabled. I ran across this another time and it was as simple as changing a (period) to a (comma). Should just do a copy and paste, less errors that way.

Thanks for your efforts,

Thomas

**Comment by Aris-t2** [comment]
"Please see attached screen shot."
Is there a new one or are you referring to those in the opening post?

**Comment by TW462** [comment]
Hi Aris-t2,

It's a separate issue. After I replied to you, I started thinking about it after looking at the screenshot attachment that it might be an FF89 (Proton) issue. It doesn't cause any problems, so I'm not going to bring up the issue again. It's not bad, having the tab sit above the main body with the same color as the background for separation. It would be great if Mozilla would stop making it so difficult to personalize/modify Firefox.

Again, thank you for all your help and input.

Thomas
